05-look-around.Rmd

```diff
@@ -0,0 +1,35 @@
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+# Look around
+
+```{r, echo=FALSE, fig.alt='We are on the fourth step of the pathway.', out.width = '80%', fig.align = 'center'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1BQxrVYdKZTbpCaF-i_q9w7s9x034lEXpQZDU-Sl09cs/edit#slide=id.gff2211b72f_1_206")
+```
+
+Now that you have successfully logged in to the Fred Hutch cluster, it's time to look around to see what compute node you have connected to, and what file storage systems are available.
+
+## Head and Compute Nodes
+
+After logging in, you are connected to the **Head Node** (alternative names include the Login Node, or the Submit Node). At Fred Hutch, head node is also called **Rhino**. The purpose of the head node is to serve as a launching pad for the user: you are encouraged to look for files you want to use for your analysis, and once you know what data and software you want to use, you perform the computation on the **Compute Nodes** (alternative names include the Worker Node). At Fred Hutch, the compute nodes are called **Gizmo**. The head node has very limited CPUs and RAM, and the nodes with the high CPU and RAM are in the compute nodes.
+
+The diagram below illustrates the relationship between the head node, compute nodes, and the shared file storage systems: you start at the login (head) node, and you can look at the files on the shared file storage system. When you know what data and software you want to use for compute, you let the compute nodes do the job. The compute nodes are also connected to the shared file storage system.
+
+![](https://docs.ycrc.yale.edu/img/cluster.png)
```

Comment:
Would be great to do a Hutch version of this image that labels the login / compute nodes as Rhino / Gizmo

This is where the images live: https://docs.google.com/presentation/d/1BQxrVYdKZTbpCaF-i_q9w7s9x034lEXpQZDU-Sl09cs/edit?usp=sharing